appserver/extras/embedded/common/bootstrap/src/main/java/org/glassfish/uberjar/UberJarMain.java

```diff
@@ -1,4 +1,5 @@
 /*
+ * Copyright (c) 2022 Contributors to the Eclipse Foundation
```

Comment:
2023?

Reply:
I did it in december, just cherrypicked, amended and pushed this year